ui_framework/src/components/flex/_flex_item.scss

```diff
@@ -1,7 +1,18 @@
-// FlexItem can be manually set to not grow if needed.
-.kuiFlexItem.kuiFlexItem--flexGrowZero {
-  flex-grow: 0;
-  flex-basis: auto;
+/**
+ * 1. Allow KuiPanels to expand to fill the item.
+ */
+.kuiFlexItem {
+  display: flex; /* 1 */
+  flex-direction: column; /* 1 */
```

Comment:
I'm not so sure this is an all the time wanted thing. I'd actually started this way, but figured making components more specific to usage was a better method then trying to cross-breed them together. I'm sure there are gonna be times where we don't want flex-item to itself have flex.

Reply:
I agree that this is a potential trouble spot... let's keep an eye on it and see if any problems arise.